src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random randomValueSupplier = new Random();
+    private final int maxValue = 101;
```

Comment:
The variable `maxValue` should be a constant. According to the Java style guide, constants should be in all upper case with underscores between words, and they should be declared as `static final`.